packages/atomic/cypress/integration/facets-v1/facet/facet.cypress.ts

```diff
@@ -106,69 +109,325 @@ describe('Facet v1 Test Suites', () => {
         });
       });
     });
+  });
 
-    describe('when selecting the "Show more" button', () => {
-      function setupSelectShowMore() {
-        setupWithCheckboxValues();
-        FacetSelectors.showMoreButton().click();
+  describe('with link values', () => {
+    function setupWithLinkValues() {
+      new TestFixture()
+        .with(addFacet({field, label, 'display-values-as': 'link'}))
+        .init();
+      cy.wait(TestFixture.interceptAliases.UA);
+    }
+
+    describe('verify rendering', () => {
+      before(setupWithLinkValues);
+
+      FacetAssertions.assertAccessibility();
+      FacetAssertions.assertDisplayValues(true);
+      FacetAssertions.assertNumberOfSelectedLinkValues(0);
+      FacetAssertions.assertNumberOfIdleLinkValues(defaultNumberOfValues);
+      FacetAssertions.assertDisplayClearButton(false);
+      FacetAssertions.assertDisplaySearchInput(true);
+    });
+
+    describe('when selecting a value', () => {
+      const selectionIndex = 2;
+      function setupSelectLinkValue() {
+        setupWithLinkValues();
+        selectIdleLinkValueAt(selectionIndex);
         cy.wait(TestFixture.interceptAliases.Search);
       }
 
       describe('verify rendering', () => {
-        before(setupSelectShowMore);
+        before(setupSelectLinkValue);
 
-        FacetAssertions.assertDisplayShowMoreButton(true);
-        FacetAssertions.assertDisplayShowLessButton(true);
-        FacetAssertions.assertValuesSortedAlphanumerically();
-        FacetAssertions.assertNumberOfIdleCheckboxValues(
-          defaultNumberOfValues * 2
-        );
+        FacetAssertions.assertAccessibility();
+        FacetAssertions.assertDisplayClearButton(true);
+        FacetAssertions.assertNumberOfSelectedLinkValues(1);
+        FacetAssertions.assertNumberOfIdleLinkValues(defaultNumberOfValues - 1);
       });
 
       describe('verify analytics', () => {
-        before(setupSelectShowMore);
+        before(setupSelectLinkValue);
+        FacetAssertions.assertLogFacetSelect(field, selectionIndex);
+      });
+
+      describe('when selecting a second value', () => {
+        const secondSelectionIndex = 0;
+        function setupSelectSecondLinkValue() {
+          setupSelectLinkValue();
+          cy.wait(TestFixture.interceptAliases.UA);
+          selectIdleLinkValueAt(secondSelectionIndex);
+          cy.wait(TestFixture.interceptAliases.Search);
+        }
+
+        describe('verify rendering', () => {
+          before(setupSelectSecondLinkValue);
+
+          FacetAssertions.assertDisplayClearButton(true);
+          FacetAssertions.assertNumberOfSelectedLinkValues(1);
+          FacetAssertions.assertNumberOfIdleLinkValues(
+            defaultNumberOfValues - 1
```

Comment:
If it's the second link value being selected shouldn't it be `defaultNumberOfValues -  2`?

Reply:
Link values are single-select, when you select one it's supposed to deselect the other 😄